script.js

```diff
@@ -0,0 +1,34 @@
+const bookContainer = document.querySelector('.bookContainer');
+const books = JSON.parse(localStorage.getItem('books')) || [];
+books.forEach((book) => {
+  bookContainer.innerHTML += `
+  <p>"${book.title}"</p>
+  <p>by ${book.author}</p>
+  <button class="removeBtn" id="${book.id}">Remove</button>
+  <hr>
+`;
+});
+const addBook = () => {
+  const formData = document.querySelectorAll('input');
+  const title = formData[0].value;
+  const author = formData[1].value;
+  if (title.trim() === '' || author.trim() === '') return;
+  const id = Date.now().toString();
+  books.push({ title, author, id });
+  localStorage.setItem('books', JSON.stringify(books));
```

Comment:
# kindly give us some more information, please

a video would be better because this side as you are saying is working 

I tried on mobile too

- [ ] added 3 books 
- [ ] removed a random one 
- [ ] added 2 books again
- [ ] removed 3 books again
I followed these steps to check

Reply:
- [ ] Can you attach a screenshot to show the date been preserved in localstorage (browser memory)  after the book is added and removed? 
- First screenshot when the book is added and is displayed on the page, second screenshot when the book is removed and the data still preserved in localstorage. 

When you do this, I think the next reviewer will approve your PR. 

Great job so far. Keep working hard.

Reply:
![image](https://user-images.githubusercontent.com/116451144/212816677-33c3eac4-a52c-41a4-bb9b-7f21c1e0ab47.png)
![image](https://user-images.githubusercontent.com/116451144/212816692-5be386cd-8b3e-470d-96de-b727da44bdfe.png)

Reply:
added two records:
![image](https://user-images.githubusercontent.com/41000844/212817451-82a9f69c-c70d-472a-8da2-efb332dbd4e2.png)
Removed first:
![image](https://user-images.githubusercontent.com/41000844/212817546-92d2e229-29de-4045-b792-ddaa53d86e75.png)